src/components/AdminPage/AdminPageLayout.js

```diff
@@ -20,6 +20,7 @@ export const SearchAndSortWrapper = styled("div")`
     display: grid;
     grid-template-columns: auto auto;
     justify-content: space-between;
+    border: 3px solid #73AD21;
```

Comment:
What is this change related to?

Reply:
Debugging some CSS stuff, again, got my git branches tangled up. Will fix and submit again.

Reply:
Got it, looks good!